tests/default.test.ts

```diff
@@ -21,14 +21,81 @@ test("Empty default", () => {
 
   expect(result).toMatchSnapshot();
 });
-test("Empty default", () => {
-  const result = subject(`
+
+test("default empty array", () => {
+  const input = `
   /**
    * The value
    *
    * @default []
    */
-`);
+`
+  const result = subject(input);
+
+  expect(result).toMatchSnapshot();
+});
+
+test("default empty object", () => {
+  const input = `
+  /**
```

Comment:
Please add these tests too.

```
  /**
   * The value
   *
   * @default "somthing"
   * @default {}
   */
```


```
  /**
   * The value
   *
   * @default {}
   * @default "somthing"
   */
```

Reply:
Is the expectation in those cases to delete the 2nd `@default` statement?

Reply:
Deleting a tag is dangerous. Maybe that is good to keep both.